src/sagemaker_pytorch_container/training.py

```diff
@@ -96,6 +96,13 @@ def train(training_environment):
         capture_error = False
         logger.info(f'capture_error is {capture_error}. Default is True')
 
+    try:
+        import torch
+
+        os.environ["SM_DLC_TORCH_VERSION"] = torch.__version__
+    except Exception:
+        pass
```

Comment:
is `pass` intentional ?

Reply:
Please follow https://github.com/aws/sagemaker-pytorch-training-toolkit/blob/master/CONTRIBUTING.md#committing-your-change

Reply:
You should catch the exception and log it as info probably.

Reply:
Yes.